admin/gs_check.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# Create original PS plot and PNG image that include these GMT features:
+# a) patterns
+# b) image
+# c) transparency
+# d) lines and annotations, including degree symbol
+#
+# Then run the new candidate gs with suitable options to create a PNG we can compare
+# using gmt compare to the original PNG that we have verified manually.
+# Run from the top gmt-dev directory
+
+if [ ! -d cmake ]; then
+	echo "Must be run from top-level gmt directory"
+	exit 1
+fi
+if [ ! -d build ]; then
+	mkdir -p build
+fi
+# Do the work in the build directory
+cd build
```

Comment:
Echo something here so that others know that the script outputs everything in the build directory.

Reply:
OK will do